pom.xml

```diff
@@ -1257,7 +1258,7 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>versions-maven-plugin</artifactId>
-        <version>2.1</version>
+        <version>2.3</version>
         <reportSets>
```

Comment:
Yes, for execution, however I did not do the equivalent of a full apache release because I don't know how to test this, but the jars build and the tests execute correctly.